src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1283,6 +1283,23 @@ ApplicationManagerImpl::GetHandshakeContext(uint32_t key) const {
   }
   return SSLContext::HandshakeContext();
 }
+
+bool ApplicationManagerImpl::CanStartProtectedService(
+    const int32_t& session_key,
+    const protocol_handler::ServiceType& type) const {
```

Comment:
@okozlovlux Passing parameter by const reference prevents for using it as lvalue in the function scope, so I prefer to keep this one as it is.